src/app/shared/services/cart.service.ts

```diff
@@ -23,14 +23,14 @@ export class CartService {
         return this.goods.find(item => item.good.id === good.id);
     }
 
-    public addToCart(good: ProductInterface): GoodInterface[] {
+    public addToCart(good: ProductInterface, count: number = 1): GoodInterface[] {
         const existedGood = this.isExistGoodInCart(good);
         if (!existedGood) {
-            this.goods.push({ good, count: 1 });
+            this.goods.unshift({ good, count });
```

Comment:
I dont like this 'unshift', with this method you have a issue with performance.
Maybe you should use 'push' and apply 'flex-direction: column-reverse' in yours styles